src/unk_020507CC.c

```diff
@@ -9,29 +9,29 @@
 typedef struct UnkStruct_020507E4_t {
     u16 unk_00[288];
     u8 unk_240[364];
-} UnkStruct_020507E4;
+} FieldEvents;
```

Comment:
Why name this struct `FieldEvents` when all its fields are still unknown?

Reply:
It was the struct returned from SaveData_GetEvents, a function I did not name

Reply:
Keep the struct named as unknown then if you don't know what it does (also that function is named wrongly since the struct has nothing to do with field events)

Reply:
We discussed a different name already; `SaveVars`, maybe? it sure looks like a similar struct to the one in HG for vars and flags to me.

Reply:
Reference: https://github.com/pret/pokeheartgold/blob/e1f0deee14d3a5fb336e662b124c795c5f02ae36/include/save_vars_flags.h#L8

Reply:
```c
typedef struct VarsFlags {
    u16 vars[288];
    u8 flags[364];
} VarsFlags;
```